tests/core/core.spec.ts

```diff
@@ -931,6 +946,17 @@ suite("@dataform/core", () => {
       ).deep.equals(["schema.operate-1"]);
       expect(graph.operations[1].queries).deep.equals(['select * from "schema"."operate-1"']);
     });
+
+    [{testConfig: TestConfigs.redshift, finalizedSchema: 'schema'}, {testConfig: TestConfigs.redshiftWithSuffix, finalizedSchema: 'schema_suffix'}].forEach(({testConfig, finalizedSchema}) => {
```

Comment:
seems bigger than line limit. ideally, prettier should be auto-formatting your code, can you look into that?

Reply:
Did speak with Elias about that, he said we use the auto format in VS code, but that didn't touch line length. I did notice we have other tests with an `array.forEach` that goes past the standard 100 character width, so left that.

I suggest we add some global format file/tool (eg. a `tslint`) and update the contributing.md too, to mention using this. WDYT?

Reply:
We do use tslint in closed-source, I'm a little surprised it's not enforced here (as part of presubmit tests).

Reply:
Will look into adding this.